sass/partials/global/variables/_typography.scss

```diff
@@ -2,9 +2,10 @@
 // Typography variables
 // http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#variables_
 
-$base-font-size: 16px;
-$base-line-height: 24px;
+$base-font-size: 17px;
+$base-line-height: 25.5px;
```

Comment:
25.5px for the line-height is just what the U.S. Web Design Standards use, which is the same as a line height of 1.5 for the body copy.